### PR TITLE
Initial commit - Making wrapper measurement changes for WaitForControlledPodsRunning

### PR DESCRIPTION
**Pull request by vamossagar12**
This PR is related to https://github.com/kubernetes/perf-tests/issues/794. The actual code changes for the wrapper measurement are in https://github.com/kubernetes/perf-tests/pull/1075 and have been merged. This PR deals mainly with the config changes.

### CODE REVIEW

File: clusterloader2/testing/load/config.yaml [hunk 1]
```diff
@@ -160,39 +160,28 @@ steps:
         objectTemplatePath: daemonset-priorityclass.yaml
 
 - name: Starting measurement for waiting for pods
-  measurements:
-  - Identifier: WaitForRunningDeployments
-    Method: WaitForControlledPodsRunning
-    Params:
-      action: start
+  Method: WaitForControlledPodsRunning
```
Comment (wojtek-t) [inline clusterloader2/testing/load/config.yaml:163]: measurements:

missing?

File: clusterloader2/testing/load/config.yaml [hunk 2]
```diff
@@ -160,39 +160,28 @@ steps:
...
+  Configs:
```
Comment (wojtek-t) [inline clusterloader2/testing/load/config.yaml:164]: What is Configs?
https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/api/types.go#L175

File: clusterloader2/testing/load/config.yaml (hunk 1 above)
Comment (vamossagar12) [reply clusterloader2/testing/load/config.yaml]: Added it. sorry missed it thinking it isn't needed.

File: clusterloader2/testing/load/config.yaml (hunk 2 above)
Comment (vamossagar12) [reply clusterloader2/testing/load/config.yaml]: configs is Instances. I took the config from the old conversation. sorry should have checked.

**Comment by wojtek-t** [comment]
@vamossagar12 - the test config doesn't parse:
https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/perf-tests/1407/pull-perf-tests-clusterloader2/1288752324595945472/

Please figure out what is wrong - I'm not going to look into this PR again until the tests are passing.

**Comment by vamossagar12** [comment]
hey @wojtek-t , I corrected the config file errors. This time the tests passed.

**Comment by wojtek-t** [comment]
/lgtm
/approve

Thanks!

**Comment by vamossagar12** [comment]
Thanks @wojtek-t , on similar lines, what other files should be modified?

**Comment by vamossagar12** [comment]
@wojtek-t , I had issued a re-test because of the failure of 1 test case. It got auto-merged. Is this expected behaviour? Sorry I am not aware of the workings of the ci-robot.

**Pull request merged**